README.md

```diff
@@ -1,144 +1,110 @@
 ```
-$ dnf install -y make qemu-system-x86 libguestfs-tools-c expect
-```
-
-
-## Build
 
-First you need to build the image:
+3. Deploy KubeVirt on it
 
 ```bash
 $ git clone https://github.com/kubevirt/demo.git
 $ cd demo
-$ make build
-```
-
-This can take a while, as a base image and several containers are getting
-downloaded an deployed inside the image.
-
-
-## Use
-
-**Note:** Use `root` to login without a password.
-
-Now that the image is completed, you can use it:
-
-```bash
 $ ./run-demo.sh
+$ ./run-demo.sh 
+KubeVirt (v0.0.1-alpha.6) demo on minikube
+- Checking kubectl version ... OK
+- Checking for minikube ... OK
+- Checking out KubeVirt ... OK
+- Deploying manifests - this can take several minutes!
+vm "testvm" created
+- Waiting for the cluster to be ready ...
+  Cluster changed, checking if KubeVirt is ready ... Not yet.
+...
+  Cluster changed, checking if KubeVirt is ready ... Yes!
+KubeVirt is now ready.
+Optional, register the virt plugin for VM access:
+  ./run-demo.sh plug
+Try: $ kubectl get vms
+
+# Run a script to verify that it operates correctly
+$ ./test.sh
+$ ./test.sh 
+README contains correct version ... OK
+VM is running ... OK
+VM serial console works ... OK
+PASS
 ```
 
-This will boot you into a virtual serial console of the VM.
-First you need to login, once you are done, you can shut it down.
-
-### `kubectl`
-
-After the deployment you can use `kubectl` to create VMs:
-
-```bash
-$ kubectl create -f /vm.json
-```
-
-To view created VMs use the following command:
+4. Start managing VMs
 
 ```bash
+# After deployment you can manage VMs using the usual verbs:
 $ kubectl get vms
-```
+$ kubectl get vms -o json
 
-### `virsh`
-
-You can also use `virsh` inside the VM to look at what is
-happening in libvirt directly:
-
-```bash
-$ virsh list --all
+# To delete: kubectl delete vms testvm
+# To create your own: kubectl create -f $YOUR_VM_SPEC
 ```
 
-### Cockpit
-
-You can also view Cockpit running inside the VM to look at the
-Kubernetes topology and the involved KubeVirt components.
-
-Just point your browser to <https://127.0.0.1:9091/kubernetes>.
+### Accessing VMs (serial console & spice)
 
+Currently you need a separate tool to access the graphical display or serial
+console of a VM, you can install it as a `kubelet` plugin using:
 
-### Kubernetes Dashboard
-
-The [kubernetes dashboard](https://github.com/kubernetes/dashboard/) is also getting deployed.
-You can access it after boot on <http://127.0.0.1:8002/ui>.
-
+```bash
+$ ./run-demo.sh plug
+KubeVirt (v0.0.1-alpha.6) demo on minikube
+- Checking kubectl version ... OK
+- Fetching and registering virtctl ... OK
 
-## Check (optional)
+# Now the plugin is ready to use
 
-If you want, then you can use `make check` to run a minimal integration test.
-This will create a VM and check if it's really getting created in libvirtd.
+# Connect to the serial console
+$ kubectl plugin virt -- console -s http://$(minikube ip):8184 testvm
```

Comment:
The -s is not so nice. We could proxy the whole traffic throught 8184. Maybe we should create a second config file based on the provided one and set this server address. Undeploy could undo that change too ...

Reply:
Stu's work should solve this, thus I'd not do a workaround here atm.